gadget/gadget.go

```diff
@@ -121,10 +121,25 @@ type Info struct {
 	KernelCmdline KernelCmdline `yaml:"kernel-cmdline"`
 }
 
+type PartialPiece string
```

Comment:
s/Piece/Property/ ?

Reply:
Good suggestion, I was struggling to find a good type name. Changed.